app/Services/AdventuresLeagueAdapter.php

```diff
@@ -153,12 +127,43 @@ private function getEntries($data, $characterId)
                     'description' => (array_key_exists(6, $data[$i])) ? $data[$i][6] : "",
                     'author_id' => Auth::id(),
                     'tier' => 0,
-                    'counted' => 0,
                 ];
-                $item = new Item($itemData);
-                $item->save();
+                Item::create($itemData);
             }
         }
         return $entries;
     }
+
+    private function getEntryData($characterId, $data, $type)
```

Comment:
since you're checking data is null later in this function, we should probably define it as default null, same with type
```suggestion
    private function getEntryData($characterId, $data = null, $type = null)
```

Reply:
done!